.github/actions/neon-project-create/action.yml

```diff
@@ -5,12 +5,12 @@ inputs:
   api_key:
     desctiption: 'Neon API key'
     required: true
-  environment:
-    desctiption: 'dev (aka captest) or staging'
-    required: true
   region_id:
     desctiption: 'Region ID, if not set the project will be created in the default region'
     required: false
+  postgres_version:
+    desctiption: 'Postgres version; default is 15'
+    required: false
```

Comment:
Could a `default: 15` field be added here, as in our `run-python-test-set` action? Presumably, we would not need `|| 15` below then.

Reply:
Ah, nice catch! I forget that we can do it :) 
I also moved default region_id to inputs default. Much cleaner now, thanks!